### ISSUE TITLE
support "using-style" joins

### ISSUE THREAD

**Issue by norbitheeviljester** [serge-rider/dbeaver#442]
Hi,
In Oracle and Postgresql there are two ways of performing a join. One of them is the using style syntax:

![image](https://cloud.githubusercontent.com/assets/6532949/15247315/bc1ee486-1913-11e6-8dc3-307b09afd7df.png)

the idea is that when the columns on which both tables are joined have the same name, then the "using-style" can be used.
I'd like to request that dbeaver's content assist supports this. When I am inside a "using( )" clause, please make the content assist provide me with a list of columns that exist in both tables simultaneously.

**Comment by serge-rider** [comment]
It seems to be correlated with #397. Although it is not an exactly duplicate.

**Comment by norbitheeviljester** [comment]
yes, I've seen the referenced issue before but it seems to be a bit more advanced from what I'm proposing. Besides the referenced feature request requires foreign keys to exist on the database in order for the new functionality to be useful. 
This is not always the case for big databases - for example in case of our databases foreign keys slow down the data modification quite significantly because of the sheer amount of data. Therefore we postpone or sometimes even skip foreign keys entirely and keep the database coherent using application logic.
The feature request I propose looks at all tables mentioned in the join and looks at the column names - no foreign keys necessary.

**Comment by serge-rider** [comment]
Agreed. Anyhow it will be resolved along with #397
